plugins/PagePerformance/lang/en.json

```diff
@@ -0,0 +1,34 @@
+{
+    "PagePerformance": {
+        "ColumnAveragePageLoadTime": "Avg. page load time",
+        "ColumnAveragePageLoadTimeDocumentation": "Average time how long it takes from requesting a page until the page is fully rendered within the browser",
+        "ColumnAverageTimeLatency": "Avg. latency time",
+        "ColumnAverageTimeLatencyDocumentation": "Average time how long it takes the response to get to the user’s device. Includes the time it takes for the request to get to the server, it takes the server to render a response, and until the first byte of that response gets back to the user’s device.",
+        "ColumnAverageTimeTransfer": "Avg. transfer time",
+        "ColumnAverageTimeTransferDocumentation": "Average time how long it takes the browser to download the response from the server.",
+        "ColumnAverageTimeDomProcessing": "Avg. DOM processing time",
+        "ColumnAverageTimeDomProcessingDocumentation": "Average time how long the browser spends loading the webpage until the user can starting interacting with it.",
```

Comment:
Be great to mention from what time this is counted. Eg from dom ready, from sending the first byte of the request to the server, from ...